README.md

```diff
@@ -219,6 +219,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [tarantool-crystal](https://github.com/vladfaust/tarantool-crystal) - Tarantool driver
 
 ## Database Tools
+ * [crecto-admin](https://github.com/Crecto/crecto-admin) - Admin dashboard for Crecto and your database
```

Comment:
Is Crecto Admin still working with latest Crystal changes?

Reply:
It seems like some updates have happened recently but i don't think crecto has had an official release since support.

Reply:
I'm glad to see that Crecto's back! However, maybe we should ask for @fridgerator's input about Crecto Admin?